.github/workflows/preview.yml

```diff
@@ -4,9 +4,9 @@ name: Preview
 on: [pull_request]
 
 jobs:
-  code-engine:
+  preview-setup:
```

Comment:
Regarding this, @Tansito was looking for a way to show a nice action message, so he may have opinions about this.

Reply:
Yeah, the idea was to try to obtain this result: `Preview / code-engine / deploy` a little example: https://github.com/Qiskit/qiskit.org/pull/2408

But I'm open to other ideas.

Reply:
something like works for me and other possible options to give you more ideas:
- `Preview / set-up / code-engine` and `Preview / tear-down / code-engine`
- `Preview / set-up / deploy` and `Preview / tear-down / clean-up`

Reply:
Another idea:

- `Preview / code-engine / deploy`
- `Preview / code-engine / cleanup`